src/plugins/safeSnap/components/HandleOutcomeUma.vue

```diff
@@ -219,11 +219,10 @@ const questionState = computed<QuestionState>(() => {
 
   if (!questionDetails.value) return 'error';
 
-  const { assertionEvent, proposalExecuted, activeProposal } =
+  const { assertionEvent, proposalExecuted, activeProposal, noTransactions } =
     questionDetails.value;
 
-  if (assertionEvent === undefined || questionDetails.value.noTransactions)
```

Comment:
this was causing all unproposed proposals to fail since there was no assertion event yet